examples/minigame.py

```
@@ -0,0 +1,46 @@
+    def Exit(self):
+        print("--------- ------ ------ Bye!")
+
+row_A = MainMenu()
+index = 0
+
+while True:
+    row_A.run(index)
+
+    match row_A.received[0]:
+
+        case '1': index = 1
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)